### DISCUSSION TITLE
37586F18-3941-4BDD-83D7-29C71009DB19

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#157] [General]
# 37586F18-3941-4BDD-83D7-29C71009DB19

MASCARA-5, also known as HD 201033 and TOI-1431, is an A-type main-sequence star about 487.79 light-years away in the constellation Cepheus. A highly irradiated gas giant planet larger than Jupiter completes an orbit around its star about once every 2.65 days

https://celestia.mobi/addon?item=37586F18-3941-4BDD-83D7-29C71009DB19

**Comment by exocritic5457** [comment]
I do have to wonder why you chose neon lights as the planet's glow but that's kind of a ubiquitous thing in some of the addons at this point.
